<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hydraharp_mainWindow</class>
 <widget class="QMainWindow" name="hydraharp_mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>513</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Simple XY Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="hydraharp_modules">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>380</height>
       </size>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="modules_layout">
       <item>
        <widget class="QCheckBox" name="connected_checkBox">
         <property name="text">
          <string>connected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="on_pushButton">
         <property name="text">
          <string>ON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="off_pushButton">
         <property name="text">
          <string>OFF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>target power:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="target_power_doubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="channel_optimizer_GroupBox">
      <property name="title">
       <string>Channel Optimizer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="channel_optimizer_layout">
       <item>
        <widget class="QFrame" name="optimizer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="optimizer_layout">
          <item>
           <layout class="QVBoxLayout" name="B1">
            <item>
             <widget class="QCheckBox" name="run_checkBox">
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="temperature_visible_checkBox">
              <property name="text">
               <string>plot temperatures</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QDoubleSpinBox" name="N_pts_doubleSpinBox">
              <property name="prefix">
               <string>N_pts: </string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="Navg_doubleSpinBox">
              <property name="prefix">
               <string>Navg: </string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>